content/src/scripts/datalayer/v1/datalayer.js

```diff
@@ -47,6 +47,20 @@
         var element = event.currentTarget;
         var componentId = getClickId(element);
 
+        // add clicked link contained by the Text Component to Data Layer
```

Comment:
In order to keep the logic of the `datalayer.js` agnostic of specific components, we might as well place this logic as part of the Text component, as we do when managing the `show` and `hide` events of the Accordion component [0].
@comanV @vladbailescu @bartoszglow WDYT?

[0] https://github.com/adobe/aem-core-wcm-components/blob/development/content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js#L367

Reply:
I agree, I think it's better.